SparkPost/2web2ui
# SA-1617: Charts localize request

### What Changed
 - Charts will separate based on number of comparison filters
 - Charts and tabs panel separated from the report options panel

### How To Test
 - Enable account UI option `allow_compare_by`
 - Go to /signals/analytics
 - Click on `Add Comparison` and  fill out form w/ at least 2 options
 - Verify that charts render multiple
 - Reopen form and clear filters
 - Verify charts render w/o label

### To Do

### Notes
 A lot of this branch is rebased on SA-1773. Will be smaller once merged
